app/assets/javascripts/admin/addon/controllers/admin-area-settings-base.js

```diff
@@ -1,13 +1,19 @@
-import { tracked } from "@glimmer/tracking";
 import Controller from "@ember/controller";
 import { action } from "@ember/object";
 
 export default class AdminAreaSettingsBaseController extends Controller {
-  @tracked filter = "";
-  queryParams = ["filter"];
+  filter = "";
+  queryParams = [
+    {
+      filter: { replace: true },
+    },
+  ];
 
   @action
   adminSettingsFilterChangedCallback(filter) {
-    this.filter = filter;
+    if (this.filter === filter) {
+      return;
+    }
+    this.set("filter", filter);
```

Comment:
What is the reason we need to change `@tracked` with regular assignment to untracked using `#set`?

Reply:
Since these are controller params I was finding `@tracked` wasn't working well, it's working OK now